pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -165,12 +166,13 @@ async def run(self):
                 await self._change_advisory_status()
                 await slack_client.say("Complete preparing microshift advisory.", slack_thread)
         except Exception as err:
-            error_message = f"Error building microshift: {err}\n {traceback.format_exc()}"
-            if assembly_type != AssemblyTypes.STREAM:
-                error_message += "\n@release-artists"
+            slack_message = f"Error preparing microshift advisory: {err}"
```

Comment:
the exception may come from build failure, preparing the advisory is only for the release build, I don't think it needs to be modified here

Reply:
Ah good catch, made a mistake

Reply:
Reverted to previous wording